client/app/app.translate.js

```diff
@@ -0,0 +1,13 @@
+import localeEN from "./locales/locale-en.json";
+import localeDE from "./locales/locale-de.json";
+
+let AppTranslate = ($translateProvider) => {
+  "ngInject";
+  $translateProvider
+  .translations("en", localeEN)
```

Comment:
I guess this is better: `registerAvailableLanguageKeys()`

```
    $translateProvider
      .translations('en', { /* ... */ })
      .translations('de', { /* ... */ })
      .registerAvailableLanguageKeys(['en', 'de'], {
        'en_*': 'en',
        'de_*': 'de'
      })
      .determinePreferredLanguage();
```
https://angular-translate.github.io/docs/#/guide/09_language-negotiation